start/sw/web-app/laravel5.md

```diff
@@ -29,6 +29,8 @@ OR, you can rip the benefit of using Fedora, and install composer directly inste
 $ sudo dnf install composer
 ```
 
+You could also choose [Apache](/start/sw/web-app/apache.html) as the web server. But it requires an additonal package, `php`, to serve PHP contents properly. After installing the package, Apache needs to be restarted to take effect.
```

Comment:
This fits better on line 18 (`#### Install Nginx Web Server`) of this file.

Reply:
Also I think all of the web servers need to be reinstalled to be able to interpret php after it's installation.

Reply:
I'll move it to line 18.
And I'm sure Apache doesn't need to be reinstalled after `php` installation.

Reply:
Sorry, I meant restarted.